src/store/d-apps/d-apps-state.ts

```diff
@@ -18,16 +17,7 @@ export interface DAppsState {
 export const dAppsInitialState: DAppsState = {
   permissions: createEntity([]),
   dappsList: createEntity([]),
-  tokensApyInfo: {
-    [KUSD_SLUG]: {
-      rate: 0,
-      link: YUPANA_LINK
-    },
-    [TZ_BTC_SLUG]: {
-      rate: 0,
-      link: KORDFI_LINK
-    }
-  }
+  tokensApyInfo: {}
```

Comment:
maybe change this to `tokensApyRates: Record<TokenSlug, number>`?  
to prevent using data that is already in store?